ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -460,6 +460,25 @@ spec:
         - name: GITHUB_TOKEN_SECRET_KEY
           value: "$(params.github_token_secret_key)"
 
+    # Add the connect page URL in github PR comment to view the test results/logs
+    - name: github-add-connect-url-comment
+      when:
+        - input: "$(tasks.certification-project-check.status)"
+          operator: in
+          values:
+            - Succeeded
```

Comment:
See my other comment in the hosted pipeline. I think we can remove this `when` condition.

Reply:
Same as above, I will remove the when condition here too.